substrate/client/network/src/request_responses.rs

```diff
@@ -207,6 +255,46 @@ impl IfDisconnected {
 		}
 	}
 }
+#[derive(Debug)]
+pub enum Event0<TRequest, TResponse, TChannelResponse = TResponse> {
```

Comment:
```suggestion
pub enum Event<TRequest, TResponse, TChannelResponse = TResponse> {
```

Reply:
I will try to move the content of `Event0` into `Event`